h2o-bindings/bin/gen_R.py

```diff
@@ -40,13 +40,20 @@ def gen_module(schema, algo, module):
             continue
         if algo == "naivebayes":
             if param["name"] == "min_sdev":
-                yield "#' @param threshold The minimum standard deviation to use for observations without enough data. "
+                yield "#' @param threshold This argument is deprecated, use `min_sdev` instead. The minimum standard deviation to use for observations without enough data. "
+                yield "#'                  Must be at least 1e-10."
+                yield "#' @param min_sdev The minimum standard deviation to use for observations without enough data. "
                 yield "#'                  Must be at least 1e-10."
                 continue
             if param["name"] == "eps_sdev":
-                yield "#' @param eps A threshold cutoff to deal with numeric instability, must be positive."
+                yield "#' @param eps This argument is deprecated, use `eps_sdev` instead. A threshold cutoff to deal with numeric instability, must be positive."
+                yield "#' @param eps_sdev A threshold cutoff to deal with numeric instability, must be positive."
```

Comment:
The following test shows an eps_sdev value of -60. Should that be updated?
https://github.com/h2oai/h2o-3/blob/master/h2o-r/tests/testdir_algos/naivebayes/runit_naivebayes_grid_cars_negative.R